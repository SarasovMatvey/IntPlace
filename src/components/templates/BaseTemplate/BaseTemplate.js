import Header from "../../organisms/Header";

function BaseTemplate({ children }) {
  return (
    <>
      <Header />
      {children}
    </>
  );
}

export default BaseTemplate;
