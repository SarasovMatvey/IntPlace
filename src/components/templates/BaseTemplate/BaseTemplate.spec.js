import BaseTemplate from "./BaseTemplate";

const setUp = props => shallow(<BaseTemplate {...props} />);

describe("BaseTemplate", () => {
  let component;

  beforeEach(() => {
    component = setUp();
  });

  it("should renders properly", () => {
    expect(component).toMatchSnapshot();
  });
});
