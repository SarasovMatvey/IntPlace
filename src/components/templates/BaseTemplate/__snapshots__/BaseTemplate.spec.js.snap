// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseTemplate should renders properly 1`] = `
<Fragment>
  <Header />
</Fragment>
`;
